keras/utils/image_utils.py

```diff
@@ -35,12 +35,12 @@
 
 if pil_image is not None:
     _PIL_INTERPOLATION_METHODS = {
```

Comment:
Please make this code block conditional on the PIL version. We cannot merge the change otherwise, in particular because the internal Google version has not been updated and does not have the `Resampling` enum.

Reply:
updated to check for the existence of the enum. let me know if a version check would be preferred instead. thanks!